README.md

```diff
@@ -1,67 +1,66 @@
-﻿# Project Reunion
+﻿# Project Reunion - Calling all Windows desktop apps!
 
-[![Feature Proposals](https://img.shields.io/github/issues/microsoft/projectreunion/feature%20proposal)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+proposal%22)
-[![Bugs](https://img.shields.io/github/issues/microsoft/projectreunion/bug)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+| Delight users with modern UI | Access new platform features | Down-level support | 
+|:--:|:--:|:--:|
+| <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-get-started.svg" width=250 alt="Modern navigation"/></br>Powerful WinUI 3 UI | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-scenarios.svg" width=250 alt="Platform logo"/></br>Localization + power status + more</br> | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-host-integration-install-configure.svg" width=250 alt="Down-level logo"/></br>Back to Windows 10 1809 |
 
+...and more, **without having to rewrite your app!**
 
-### What is Project Reunion?
-Project Reunion is our vision for unifying and evolving the Windows developer platform to make it easier
-to build great apps that work across all the Windows 10 versions and devices people use.
+* Use your current installer *(no requirement to use MSIX, but there are [reliability/security benefits to using MSIX](https://docs.microsoft.com/windows/msix/overview#key-features)!)*
+* 100% additive *(only add what you need, leave the rest of your app as-is)*
+* Works with Win32, WPF, WinForms, and UWP apps
 
-Project Reunion makes it easier to build a great Windows app by providing a unified platform for new 
-and existing Win32 and UWP apps. It will unify access to existing Win32 and UWP APIs and make them 
-available decoupled from the OS, via tools like NuGet. 
+Plus, **it works down-level** to build 17763 (version 1809/October 2018 Update) of Windows 10!

```

Comment:
Is there a "support lifecycle" link that we could put here so folks can follow that to see what this means?

Reply:
Hmm don't think we have that yet, but Dina's working with the CSS team on creating a SUPPORT.md file so maybe we'll be able to link to that in the future

Reply:
Thanks Rafael, good point. Today, all features in Reunion work down to build 17763, but in the future there inevitably are going to be new Windows platform features that are tied to the OS or Shell or hardware that simply can't come down (for example, new features in Action Center that rely on UI changes within Action Center). When possible, all features will be backwards-compatible. Features that aren't should be the rare exception.